knowledge-base/kb-chat-focus-textbox-on-load.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Setting focus to RadChat on load
```

Comment:
The 'title', 'description' and 'page_title' should contain different wording in order to optimize the search engines. You can The first two (title and description) are good, but you can think how to improve the 'page_title'. 

For example, you can replace the "Setting" word with something similar ("providing", "giving", etc.). And also, you can replace the word "load" with "initialize".